libkbfs/config_local.go

```diff
@@ -935,26 +978,16 @@ func (c *ConfigLocal) EnableJournaling(
 		return err
 	}
 
-	const backpressureMinThreshold = 0.5
-	const backpressureMaxThreshold = 0.95
-	// Cap journal usage to a quarter of the free space.
-	const journalByteLimitFrac = 0.25
-	// Set the absolute journal byte limit to 50 GiB for now.
-	const journalByteLimit int64 = 50 * 1024 * 1024 * 1024
-	// Set the absolute journal file limit to 1.5 million for now.
-	const journalFileLimit int64 = 1500000
-	bdl, err := newBackpressureDiskLimiter(
-		log, backpressureMinThreshold, backpressureMaxThreshold,
-		journalByteLimitFrac, journalByteLimit, journalFileLimit,
-		defaultDiskLimitMaxDelay, journalRoot)
-	if err != nil {
-		return err
+	if c.DiskLimiter() == nil {
```

Comment:
Can we just do `MakeDiskLimiter` in whatever initializes `ConfigLocal`? Then we wouldn't have to do this. Note that this function can technically be called from multiple goroutines, anyway, so this would be racy.